server/services/store/sqlstore/board_insights.go

```diff
@@ -0,0 +1,134 @@
+package sqlstore
+
+import (
+	"database/sql"
+	"fmt"
+
+	"github.com/mattermost/focalboard/server/model"
+
+	sq "github.com/Masterminds/squirrel"
+
+	"github.com/mattermost/mattermost-server/v6/shared/mlog"
+)
+
+func (s *SQLStore) getTeamBoardsInsights(db sq.BaseRunner, teamID string, duration string) ([]*model.BoardInsight, error) {
+	/**
+	Some squirrel issues to note here are
+	1. https://github.com/Masterminds/squirrel/issues/285 - since we're using 1+ sub queries. When placeholders are counted for second query, the placeholder names are repeated.
+		This is the reason to not use conditional operators in Where clauses which would eventually parametrize the variables.
+	*/
+
+	boardsHistoryQuery := s.getQueryBuilder(db).Select("boards.id", "boards.title", "boards.icon",
+		"count(boards_history.id) as count", "boards_history.modified_by", "boards.created_by").
+		From(s.tablePrefix + "boards_history as boards_history").
+		Join(s.tablePrefix + "boards as boards on boards_history.id = boards.id").
+		Where(fmt.Sprintf("boards_history.insert_at > %s and boards.team_id = '%s' and boards_history.modified_by != 'system' and boards.delete_at = 0",
```

Comment:
Did you meant `boards.delete_at != 0",`? (`!=` vs `=`)

Reply:
My assumptions were wrong. I'd wait for @mkraft 's comments again. 

Deleting a board isn't a soft-delete.